WordPress/Classes/ViewRelated/Reader/Tab Navigation/ReaderTabViewController.swift

```diff
@@ -137,11 +139,17 @@ extension ReaderTabViewController {
         navigationController?.present(selectInterestsViewController, animated: true, completion: nil)
 
         selectInterestsViewController.didSaveInterests = { [unowned self] in
+            self.selectedInterestsVisited()
+
             self.readerTabView.selectDiscover()
 
             self.selectInterestsViewController.dismiss(animated: true)
         }
     }
+
+    private func selectedInterestsVisited() {
```

Comment:
I was a bit confused by the naming of this, perhaps we can add an action word to indicate what it's doing?
```suggestion
    private func markSelectedInterestsVisited() {
```

Reply:
Done in a4169022583b6c8fbbe18d95b17df8d632d740ff